packages/core/src/modules/vc/W3cCredentialServiceOptions.ts

```diff
@@ -181,6 +181,8 @@ interface W3cVerifyPresentationOptionsBase {
 
 export interface W3cJwtVerifyPresentationOptions extends W3cVerifyPresentationOptionsBase {
```

Comment:
I had to think for a bit on what I think of this API. And I think it may be a bit confusing. Especially since `correlationId` is something internal to sphereon libraries, and proofRecordId it's not obvious this is for DIDcomm.

So..  what if we make it a `verificationContext`, which is optional, and has a unified interface that can be reused.

```
interface VerificationContext {
  /**
   * The `id` of the `ProofRecord` that this verification is bound to.
   */
  didcommProofRecordId?: string

  /**
   * The `id` of the `OpenId4VcVerificationSessionRecord` that this verification is bound to.
   */
   openId4VcVerificationSessionId?: string
}
```

I think we could also extract the `trustedCertificates`, but it would be good to also add these, as then you can pass them when calling the top-level APIs (w3c verification) (i think Sd-JWT is already possible right?)

And we will only call the get trusted certificates method if you didn't provide certificates (so if you call sdJWt.verify with certificates we won't call it). But otherwise we will

Reply:
We can also add the already given trustedCertificates here so it's the choice by the user to combine it with the hook ones or completely override them